src/stylesheets/components/_footer.scss

```diff
@@ -309,9 +343,16 @@
         justify-content: center;
         margin-right: units(0.5);
         margin-left: units(-0.5);
+
+        @media (forced-colors: active) {
+          background-color: buttonText;
```

Comment:
You can set `!important` and remove the styles on `line 375`.

Reply:
Resolved in [c1dae62](https://github.com/uswds/uswds/pull/4528/commits/c1dae62dac19c1f422211a5695d21d81ad914c57)